content/en/blog/_posts/2022-12-13-host-process-containers-ga/index.md

```diff
@@ -0,0 +1,122 @@
+
+HostProcess and Linux privileged containers enable similar scenarios but differ
+greatly in their implementation (hence the naming difference). HostProcess containers
+have their own [PodSecurityContext](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.25/#windowssecuritycontextoptions-v1-core) fields.
+Those used to configure Linux privileged containers **do not** apply. Enabling privileged access to a Windows host is a
+fundamentally different process than with Linux so the configuration and
+capabilities of each differ significantly. Below is a diagram detailing the
+overall architecture of Windows HostProcess containers:
+
+{{< figure src="hpc_architecture.svg" alt="HostProcess Architecture" >}}
+
+Two major features were added prior to moving to stable: the ability to run as local user accounts, and
+a simplified method of accessing volume mounts. To learn more, read
+[Create a Windows HostProcess Pod](/docs/tasks/configure-pod-container/create-hostprocess-pod/).
+
+## HostProcess containers in action
+
+Kubernetes SIG Windows has been busy putting HostProcess containers to use - even before GA!
+They've been very excited to use HostProcess containers for a number of important activities
+that were a pain to perform in the past.
+
+Here are just a few of the many use use cases with example deployments:
+
+- [CNI solutions and kube-proxy](https://github.com/kubernetes-sigs/sig-windows-tools/tree/master/hostprocess/calico#calico-example)
+- [windows-exporter](https://github.com/prometheus-community/windows_exporter/blob/master/kubernetes/windows-exporter-daemonset.yaml)
+- [csi-proxy](https://github.com/kubernetes-sigs/sig-windows-tools/tree/master/hostprocess/csi-proxy)
+- [Windows-debug container](https://github.com/jsturtevant/windows-debug)
+- [ETW event streaming](https://github.com/kubernetes-sigs/sig-windows-tools/tree/master/hostprocess/eventflow-logger)
+
+## How do I use it?
+
+A HostProcess container can be built using any base image of your choosing, however, for convenience we have
+created [a HostProcess container base image](https://github.com/microsoft/windows-host-process-containers-base-image).
+This image is only a few KB in size and does not inherit any of the same compatibility requirements as regular Windows
+server containers which allows it to run on any Windows server version.
+
+To use that Microsoft image, put this in your `Dockerfile`:
+
+```dockerfile
+FROM mcr.microsoft.com/oss/kubernetes/windows-host-process-containers-base-image:v1.0.0
+```
+
+You can run HostProcess containers from within a
+[HostProcess Pod](/docs/concepts/workloads/pods/#privileged-mode-for-containers).
+
+To get started with running Windows containers,
+see the general guidance for [deploying Windows nodes](/docs/setup/production-environment/windows/).
+If you have a compatible node (for example: Windows as the operating system
+containerd v1.7 or later), you can deploy a Pod with one
```

Comment:
```suggestion
f you have a compatible node (for example: Windows as the operating system
with containerd v1.7 or later as the container runtime), you can deploy a Pod with one

Reply:
@aravindhp I think there is a typo in your suggestion

Reply:
Sorry about that @sftim 

```suggestion
If you have a compatible node (for example: Windows as the operating system
with containerd v1.7 or later as the container runtime), you can deploy a Pod with one